scheduler/conf.c

```diff
@@ -558,6 +558,17 @@ cupsdReadConfiguration(void)
 
   cupsdDeleteAllListeners();
 
+ /*
+  * Allocate Listeners array
+  */
+  Listeners = cupsArrayNew(NULL, NULL);
```

Comment:
Minor nit - add a blank line after the comment...

Reply:
Resolved, thanks!